dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -30,6 +30,27 @@ Parameters:
     Default: 'false'
     Description: "If false, expect target ActiveGate to have a self-signed SSL certificate and do not verify its validity"
     Type: String
+  UseExistingActiveGate:
+    Type: String
+    AllowedValues:
+      - 'true'
+      - 'false'
+  DynatracePaasToken:
+    Description: Dynatrace PaaS token
+    Type: String
+    NoEcho: true
+  TenantIdNewAGonly:
+    Type: String
+    Default: "not_provided"
+    Description: "Needed to construct full target-url path in scenario with new AG deployment (with existing AG option, target-url is already provided by DynatraceEnvironmentUrl)"
```

Comment:
I got lost, so just to make sure I got this right:
`--use-existing-ag=true --target-url=AG_URL`
< works as it used to

`use-existing-ag=false --target-url=TENANT_URL`
< extract TenantId / TenantIdNewAGonly
< TenantId to calculate AG endpoint: https://${EC2ActiveGateIP}:9999/e/${TenantId}"

If yes then trying to think of something to make it clearer, not sure if that would do, or is still the same:
`Needed in scenario with new AG deployment to calculate an AG endpoint: https://${EC2ActiveGateIP}:9999/e/${TenantId} for the proper tenant`

Reply:
I am not sure I understand this notation of yours 100% but I think you are correct.

Why this `for the proper tenant`? This part I do not understand also

Reply:
If you go with newAG then you need to provide TenantId (bash script extracts this from TenantUrl)

Reply:
Maybe this last part in parens is confusing? I think it might be looking at it right now

Reply:
Yeap, I'd get rid of this part: `(with existing AG option, target-url is already provided by DynatraceEnvironmentUrl)`

> Why this for the proper tenant? This part I do not understand also

I meant that we would know AG address itself (https://${EC2ActiveGateIP}:9999), but we need also the path for the tenantId on this AG (https://${EC2ActiveGateIP}:9999**/e/${TenantId}** for the proper tenant)

Reply:
Rewritten this description, please check if it is good enough now